scripts/clangw

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
```

Comment:
Scary! I might personally prefer the C++ project, but that's because Bash is really scary code to write and validate.

Reply:
That's fair enough.